### DISCUSSION TITLE
[Feature Request] Overwrite subject - read from file (batch)

### DISCUSSION THREAD

**Discussion by scofano** [AIrjen/OneButtonPrompt#42] [Ideas]
It would be great to read the Overwrite subject parameter from a file or to be able to input a list.

**Comment by AIrjen** [comment]
Hi, I converted this to a discussion.

This is a good idea and relativly simple to implement. I will pick this up for the next upcoming release. Hopefully I have some time this weekend.

Just to note, you can sort of already to this with the current implementation. 
You could: 
1. create a replacement file and use the related OneButtonPrompt wildcard as the subject
2. Use Dynamic Prompts extension, and use a Dynamic Prompts wildcard

See documentation here:
https://github.com/AIrjen/OneButtonPrompt/blob/main/user_guides/custom_files.md
https://github.com/AIrjen/OneButtonPrompt/blob/main/user_guides/wildcards_and_dynamic_prompts_extension.md

However, it would be nice to have a single purpose file for this, and its own -wildcard-.

**Reply by AIrjen** [reply]
This is in the latest release. Thanks, this is a good suggestion 😊

In the user_files folder, you can place a file called custom_subjects.csv . If you use this -subjectfromfile- wildcard, it will select a random value from this file. Suggested use is on the Overwrite Subject field.

**Reply by scofano** [reply]
Thanks a lot!
